src/tests/issues.targets

```diff
@@ -1184,14 +1184,8 @@
         <ExcludeList Include="$(XunitTestBinBase)/Loader/classloader/StaticVirtualMethods/InterfaceVariance/**">
             <Issue>https://github.com/dotnet/runtime/issues/88689</Issue>
         </ExcludeList>
-        <ExcludeList Include="$(XunitTestBinBase)/Regressions/coreclr/GitHub_85240/test85240/**">
-            <Issue>https://github.com/dotnet/runtime/issues/71095</Issue>
```

Comment:
The runtime test `test85240` is still failing: https://dev.azure.com/dnceng-public/public/_build/results?buildId=568219&view=logs&j=28035387-23c9-50dd-d519-66c274947a2c&t=ea523488-181f-5d56-a79f-8dbe85d021ad&l=34
I think the reason for this is that Mono does not handle fptr type equality and compatibility.
I suggest we leave the test disabled, but update the reason to: https://github.com/dotnet/runtime/issues/90308

Reply:
Fixed. Thank you!